bokehjs/src/lib/models/ui/ui_element.ts

```diff
@@ -91,7 +91,7 @@ export abstract class UIElementView extends DOMComponentView {
   }
 
   public *_classes(): Iterable<string> {
-    yield `bk-${this.model.type}`
+    yield `bk-${this.model.type.replace(/\./g, "__")}`
```

Comment:
Why not just `-`

Reply:
+1, I don't think there's much risk of collision.

Reply:
+1 here on using "-".